fairworkflows/fairstep.py

```diff
@@ -32,23 +32,31 @@ class FairVariable:
         uri: Optionally pass a uri that the variable is referred to, the variable name will be
             automatically extracted from it. This argument is usually only used when we extract a
             variable from rdf)
-        type: The type of the variable (i.e. int, str, float etc.)
+        type: The type of the variable (i.e. int, str, float etc.
     """
-    def __init__(self, name: str = None, type: str = None, uri: str = None):
+    def __init__(self, name: str = None, computational_type: str = None, types = None, uri: str = None):
```

Comment:
I would have a slight preference for naming `types` `semantic_types`. Although it's a bit more verbose, it is more explicit.

Reply:
I agree entirely. It will be much clearer that way. I've made the changes now.